modules/cnf-configuring-kubelet-nro.adoc

```diff
@@ -41,11 +41,16 @@ spec:
       memory: "512Mi"
     topologyManagerPolicy: "single-numa-node" <5>
 ----
-<1> Adjust this label to match the `machineConfigPoolSelector` in the `NUMAResourcesOperator` CR.
+<1> Ensure this label matches the `machineConfigPoolSelector` in the `NUMAResourcesOperator` CR configured later in "Creating the NUMAResourcesOperator custom resource".
 <2> For `cpuManagerPolicy`, `static` must use a lowercase `s`.
 <3> Adjust this based on the CPU on your nodes.
 <4> For `memoryManagerPolicy`, `Static` must use an uppercase `S`.
 <5> `topologyManagerPolicy` must be set to `single-numa-node`.
++
+[NOTE]
+====
+For hosted control plane clusters, the `machineConfigPoolSelector` does not have any functional effect. Node association is instead determined by the specified `nodePool`.
```

Comment:
```suggestion
For hosted control plane clusters, the `machineConfigPoolSelector` setting does not have any functional effect. Node association is instead determined by the specified `NodePool` object.
```
In the hosted control planes documentation, the capitalization is usually `NodePool`.